test/vcr_cassettes/services/data_centers/change_validators_data_center_test/no_name.yml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
`no_name` cassette, is it used somewhere? If so, maybe we should change its name

Reply:
I'll remove that